### PR TITLE
Add initial rails configuration.

### PR DESCRIPTION
**Pull request by karlcloudy**
## Description
Initial Rails and Docker configuration added to the mySMS project.

## Breakdown of Commits
Commit 1 : 74a940f85ea839fc07610144c047c474fc0a1441
- Added initial rails structure & configuration.
- Added Docker configuration for running app.

#### TODO Checklist
* [X] create a personal GitHub repo
* [X] create a develop branch and set as the Base branch of the repo
* [X] clone repo locally
* [X] create first feature branch task_1_add_initial_rails_config
* [X] add the base Rails app files, commit and push
* [X] add docker setup (if applicable)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
+source 'https://rubygems.org'
+
+gem 'rails', '~> 3.2.0'
+
+# Bundle edge Rails instead:
+# gem 'rails', :git => 'git://github.com/rails/rails.git'
+
+gem 'mysql2', '~> 0.3.10'
```
Comment (Luke-kb) [inline Gemfile:8]: might be best to use version 0.3.20 as that is the version that JRPlus uses

**Review by Luke-kb — CHANGES_REQUESTED** [review]
Just one comment re gemfile, but other than that looks good 👍

File: Gemfile (hunk 1 above)
Comment (karlcloudy) [reply Gemfile:8]: Discussed in Slack - version that is being used by rb here is 0.3.21 (Plus has the version specified as _exactly_ 0.3.20 as mentioned). Decided that as patch releases are supposed to be backwards compatible with code targeting previous patch levels of the same major.minor version, that we'll keep it this way unless problems are uncovered in the future.

**Review by Luke-kb — APPROVED** [review]
+1 👍

**Pull request merged**